### PR TITLE
feat: Add endpoint `/accounts/{accountId}/proxy-info`

### PR DESCRIPTION
**Pull request by TarikGul**
## Summary

Add the following endpoint: `/accounts/{accountId}/proxy-info`. The following endpoint returns the proxy information from `api.query.proxy.proxies(<account_id>)`.

### Response

```
{
    at: {
        hash: string,
        height: string,
    },
    delegatedAccounts: Vec<PalletProxyProxyDefinition>,
    depositHeld: u128
}
```

### CODE REVIEW

File: docs/src/openapi-v1.yaml [hunk 1]
```diff
@@ -367,6 +367,45 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
+  /accounts/{accountId}/proxy-info:
+    get:
+      tags:
+      - accounts
+      summary: Get proxy account information.
+      description: Returns information about a proxy account. This will include 
+        delegated accounts, and deposits held.
```
Comment (bee344) [inline docs/src/openapi-v1.yaml:376]: Just a small typo here:
```suggestion
        delegated accounts and deposits held.
```

**Comment by bee344** [comment]
Shouldn't the copyright be `2017-2024` now?

File: docs/src/openapi-v1.yaml (hunk 1 above)
Comment (TarikGul) [reply docs/src/openapi-v1.yaml]: fixed

**Comment by TarikGul** [comment]
@bee344 Fixed the copyright, and the docs nit

**Review by bee344 — APPROVED** [review]

File: docs/src/openapi-v1.yaml [hunk 2]
```diff
@@ -367,6 +367,45 @@ paths:
...
+      - name: accountId
+        in: path
+        description: SS58 address of the account.
+        required: true
+        schema:
+          type: string
+          format: SS58
+      - name: at
+        in: query
+        description: Block at which to query asset approval info for the
```
Comment (Imod7) [inline docs/src/openapi-v1.yaml:388]:
```suggestion
        description: Block at which to query proxy info for the
```

**Review by Imod7 — APPROVED** [review]
Super fast & clean implementation! 💯

**Pull request merged**